pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/controller/PinotControllerTransport.java

```diff
@@ -45,6 +46,12 @@ public class PinotControllerTransport {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(PinotControllerTransport.class);
 
+  private static final long READ_TIMEOUT_MS = 60000L;
+
+  private static final int CONNECT_TIMEOUT_MS = 2000;
+
+  private static final int HANDSHAKE_TIMEOUT_MS = 2000;
```

Comment:
Can we also add a way to configure these timeouts with jdbc properties. Users may want to change these values.

Reply:
yes I can add this as jdbc properties, I will keep the defaults low (at 2000), ok?

Reply:
Yep, that works!

Reply:
I added configuration properties, should we document them?